saleor/graphql/meta/mutations.py

```diff
@@ -32,7 +36,25 @@ def _save_instance(instance, metadata_field: str):
     except FieldDoesNotExist:
         pass
 
-    instance.save(update_fields=fields)
+    try:
+        instance.save(update_fields=fields)
+    except DatabaseError as e:
+        msg = (
+            "Cannot update metadata for instance: %s. "
+            "Possible updating not existing object. "
+            "Details: %s.",
+            instance,
+            str(e),
+        )
+        logger.warning(msg)
```

Comment:
You wrote: `Possible updating not existing object.`. What kind of different situation can cause this issue? If missing object is the only one, then we could return code `NOT_FOUND` in `ValidationError`

Reply:
I am not sure about the other options tbh, thats why I am putting `e` in the msg to track it in the future.

Reply:
[fixed](https://github.com/saleor/saleor/pull/9225/commits/237af39e573412557186f5d8febf2a63e845d270)